torchtext/experimental/vectors.py

```diff
@@ -0,0 +1,60 @@
+import csv
+import torch
+from torch import Tensor
+import torch.nn as nn
+
+
+def vectors_from_csv_file(file_like_object, unk_tensor=None):
```

Comment:
nit: you probably want "from_file_object" instead of "from_csv_file" or such

Reply:
Done